### DISCUSSION TITLE
Welcome to DevOps Discussions!

### DISCUSSION THREAD

**Discussion by AryanSharma9917** [Co-Contribute-OpenSource/DevOps-practices#16] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AryanSharma9917** [comment]
Hey @Codecollab-go,I guess you're thinking of making this repo for Hacktoberfest contributions only am I correct?
If is that so then you'll need to add some more to it like making the README.md more attractive and explainable.

**Reply by Codecollab-go** [reply]
Yes, I guess you're correct will work on it for sure!! Thanks for you help
